File: examples/interpolator/interpolator.py
from MESH import Interpolator, SimulationPlanar

interpolator = Interpolator( ((3.0, (14.2, 32)), (5.4, (4.6, 10)), (5.7, (42.7, 20)),(8.0, (35.2, 40))) )
data = ()
for i in range(30, 81, 1):
  eps = interpolator.Get(i * 0.1)
  data += ((i*0.1, eps),)

print data
s = SimulationPlanar()
s.AddMaterial( "test", data )

File: examples/interpolator/interpolator.lua
interpolator = Interpolator.new({3.0, 5.4, 5.7, 8.0},
	{{14.2, 32}, -- x, and list of y values
	 {4.6, 10},
	 {42.7, 20},
	{35.2, 40}}
	)

omega = {}
epsilon = {}
for x = 3, 8, 0.1 do
	y = interpolator:Get(x)
  table.insert(omega, x)
  table.insert(epsilon, y)
end

s = SimulationPlanar.new()
s:AddMaterial("test", omega, epsilon)